config/staging/config.toml

```diff
@@ -0,0 +1 @@
+baseURL = "https://www-staging.pulumi.com/"
```

Comment:
I realize staging isn't long for this world, but we can delete this file when we make `master` deploy to `production`. For now, this maintains the status-quo.